### ISSUE TITLE
Command "go-anywhere" Fails

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#4072]
This is a quicksearch command.  It may be a Qt6 vs Qt5 matter.  Error message:
```
Traceback (most recent call last):
  File "c:\Tom\git\leo-editor\leo\core\leoKeys.py", line 2472, in callAltXFunction
    func(event)
  File "c:\Tom\git\leo-editor\leo\plugins\quicksearch.py", line 186, in find_popout_f
    def showmarks(event: LeoKeyEvent) -> None:
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: arguments did not match any overloaded call:
  setGeometry(self, a0: QRect): argument 1 has unexpected type 'float'
  setGeometry(self, ax: int, ay: int, aw: int, ah: int): argument 1 has unexpected type 'float'
```

```
Leo 6.8.2-devel, devel branch, build aa1c30137e
2024-09-26 03:33:39 -0500
Python 3.12.3, PyQt version 6.7.2
Windows 10 AMD64 (build 10.0.19045) SP0
```

**Comment by tbpassin** [comment]
I was able to fix these problems and get the command working again. It needed integers for the width and height of the popup window but expressions like `wid / 2` can produce floats.  The fix is either `int(wid/s)` or `wid//2`, e.g.,"

`        w.setGeometry(wid // 2, 100, wid // 2, 500)`

The vertical position also needs an offset instead of being zero.  I used a value of 100, as shown in the line above.

The other fix is a change in some enumerations between Qt5 and Qt6.  The focus line needs to be:

```        
         w.setFocus(Qt.FocusReason.OtherFocusReason)
                        ^^^^^^^^^
```

These two changes make the command work .... but .... the resulting window for entering the search string is almost unreadable because there are no borders delineating the fields and the widget.  It blends into the other UI elements.  A small change can turn the widget into a free-floating window and this is much  better, but then it is completely unstyled so it doesn't match the theme in effect. This could be fixed with a bit more work, and I think having the widget free-floating makes more sense (as a UI element) than having it embedded in the main splitter.

OTOH, maybe we should retire this command.  No one has used it since the switch to Qt6, and it doesn't seem to bring much value to the quicksearch experience. I didn't even know about the command and I doubt that many Leo users do.
